packages/libcrypto/libcrypto.spec

```diff
@@ -18,11 +18,20 @@ Source0: https://github.com/aws/aws-lc/archive/AWS-LC-FIPS-%{version}/aws-lc-AWS
 # git clone https://github.com/aws/aws-lc
 # cd aws-lc
 # git checkout origin/fips-2024-09-27
-# git format-patch --no-numbered AWS-LC-FIPS-3.0.0..
+# git format-patch --start-number=1001 AWS-LC-FIPS-3.0.0..
```

Comment:
We should keep the `--no-numbered` flag, otherwise folks whose `git` isn't configured to use `--no-numbered` by default may regenerate the patches with the numbered subjects.